deploy/yamls/zfs-driver.yaml

```diff
@@ -776,20 +776,18 @@ spec:
               mountPath: /plugin
             - name: device-dir
               mountPath: /dev
-            - name: encr-keys
```

Comment:
q_: what is the reason for removing this? is this not being used?

Reply:
No, this is not used.